MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1187 @@
+        {
+            dataFormSwitchItem. OnColor = Colors.Black;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Combo Box editor
+
+In the Combo Box editor, the [SfComboBox](https://help.syncfusion.com/maui/combobox)  will be loaded.
+
+#### Changing the ItemsSource of combo box
+
+By default, the `ItemsSource` for SfComboBox is auto-generated for enum types and collection type properties. For other types, you can set the `ItemsSource` by using the `IDataFormSourceProvider`.
+
+#### Using IDataFormSourceProvider
+
+{% tabs %}
+{% highlight C# %}
+
+    public string Country { get; set; }
+    
+    public class DataFormItemsSourceProvider : IDataFormSourceProvider
+    {
+        public object GetSource(string sourceName)
+        {
+            
+            if (sourceName == "Country")
+            {
+                List<string> list = new List<string>()
+                {
+                    "India",
+                    "USA",
+                    "Japan"
+                };
+                return list;
+            }
+            return new List<string>();
+        }
+    }
+    dataForm.ItemsSourceProvider = new DataFormItemsSourceProvider();
+    dataForm.RegisterEditor("Country", DataFormEditorType.ComboBox);
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using GenerateDataFormItem event
+
+You can also set the ItemsSource for combo box editor by using the `GenerateDataFormItem` event in the `DataFormComboBoxItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name" && e.DataFormItem is DataFormComboBoxItem comboBoxItem)
+        {
+             List<string> list = new List<string>()
+            {
+                "Home",
+                "Food",
+                "Utilities",
+                "Education"
+            };
+            comboBoxItem.ItemsSource = list;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+#### Enabling editing mode in combobox
+
+By default, in the combobox editor, you can select a needed item by scrolling the whole list of items.
+
+You can filter a particular item in the list items by using the `IsEditable` property, this property enables you to type a list item in the combo box editor.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "City" && e.DataFormItem is DataFormComboBoxItem comboBoxItem)
+        {
+           comboBoxItem.IsEditable = true;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Searching words that starts with the input text
```

Comment:
Text search mode definitions use as like in auto complete editor

Reply:
updated